CHANGELOG.md

```diff
@@ -35,37 +35,37 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## Unreleased
+## [v11.0.0-rc2] - 2023-01-24
```

Comment:
@fedekunze this should still say Unreleased right?

Reply:
I'd keep it as is